x-pack/plugins/enterprise_search/server/routes/enterprise_search/crawler/crawler_multiple_schedules.ts

```diff
@@ -0,0 +1,102 @@
+import { createError } from '../../../utils/create_error';
+import { elasticsearchErrorHandler } from '../../../utils/elasticsearch_error_handler';
+
+export function registerCrawlerMultipleSchedulesRoutes({ router, log }: RouteDependencies) {
+  router.post(
+    {
+      path: '/internal/enterprise_search/indices/{indexName}/crawler/custom_scheduling',
+      validate: {
+        params: schema.object({
+          indexName: schema.string(),
+        }),
+        body: schema.mapOf(
+          schema.string(),
+          schema.object({
+            name: schema.string(),
+            interval: schema.string(),
+            enabled: schema.boolean(),
+            configuration_overrides: schema.object({
+              max_crawl_depth: schema.maybe(schema.number()),
+              sitemap_discovery_disabled: schema.maybe(schema.boolean()),
+              domain_allowlist: schema.maybe(schema.arrayOf(schema.string())),
+              sitemap_urls: schema.maybe(schema.arrayOf(schema.string())),
+              seed_urls: schema.maybe(schema.arrayOf(schema.string())),
+            }),
+          })
+        ),
+      },
+    },
+    elasticsearchErrorHandler(log, async (context, request, response) => {
+      const { client } = (await context.core).elasticsearch;
+      const { params, body } = request;
+      console.log('Inside proxy');
+      console.log(body);
+      const postCustomSchedulingResult = await postCrawlerCustomScheduling(
+        client,
+        params.indexName,
+        body
+      );
+      console.log('Custom schedulleeeeeeeee');
+      console.log(postCustomSchedulingResult);
+      return response.ok({ postCustomSchedulingResult });
+    })
+  );
+
+  router.get(
+    {
+      path: '/internal/enterprise_search/indices/{indexName}/crawler/custom_scheduling',
+      validate: {
+        params: schema.object({
+          indexName: schema.string(),
+        }),
+      },
+    },
+    elasticsearchErrorHandler(log, async (context, request, response) => {
+      const { client } = (await context.core).elasticsearch;
+      try {
+        const { params } = request;
+        const customScheduling = await fetchCrawlerCustomSchedulingByIndexName(
+          client,
+          params.indexName
+        );
+        console.log(customScheduling);
+        return response.ok({
+          body: customScheduling,
+          headers: { 'content-type': 'application/json' },
+        });
+      } catch (error) {
+        if ((error as Error).message === ErrorCode.DOCUMENT_NOT_FOUND) {
+          return createError({
+            errorCode: (error as Error).message as ErrorCode,
+            message: i18n.translate(
+              'xpack.enterpriseSearch.server.routes.fetchCrawlerMultipleSchedules.documentNotFoundError',
+              {
+                defaultMessage: 'Crawler data could not be found.',
+              }
+            ),
+            response,
+            statusCode: 409,
```

Comment:
Why do we return 409 conflict for a document not found?

Reply:
Ahh copy-paste bug :) will fix